### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by ernestkorir**
1.  Project setup
2.  Install Linters
3.  Setup web pack
4.  Create the project structure

### CODE REVIEW

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @ernestkorir,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻

- [x] kindly work on project setup I was not able to run your project.
Screenshot 2022-11-15 at 11 32 04

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ernestkorir** [comment]
> 

Thanks for noting that.I have fix and i guess it's now running well

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @ernestkorir, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not running, `npm install` does nt setup the project kindly fix it

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ernestkorir** [comment]
> Hi @ernestkorir,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that is not running, `npm install` does nt setup the project kindly fix it
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

@Munsa1 Kindly I think it would be appropriate to use `npm start` . With that, i have fix that issue, and working fine on my end

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<!doctype html><html><head><meta charset="utf-8"/><title>Webpack App</title><script defer="defer" src="main.js"></script></head><body><main><section><h1>Leaderboard</h1><div id="titleRefresh"><h3 class="">Recent scores</h3><button class="">Refresh</button></div><ul style="list-style: none"><li class="nameScore"><p>Name:</p><p>100</p></li><li class="nameScore"><p>Name:</p><p>20</p></li><li class="nameScore"><p>Name:</p><p>50</p></li></ul></section><section><form action="#" id="form"><h2>Add your score</h2><input name="" id="" placeholder="Your name" style="border: solid"> <input type="number" name="" id="" placeholder="Your score" style="border: solid"> <button type="submit" style="border: solid">Submit</button></form></section></main></body></html>
```
Comment (Felix45) [inline dist/index.html:1]: :heavy_check_mark: Leaderboard: project setup is implemented as stated in the project requirements. :clap: :clap: 

Screenshot from 2022-11-16 00-58-17

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @ernestkorir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] All linter checks are setup correctly and are passing :white_check_mark: 
- [ ] Leaderboard: set up project is nicely done :white_check_mark: 
- [ ] Git flow is correctly used :white_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**